### PR TITLE
Identity V3: Get Policy

### PR DESCRIPTION
**Pull request by zhangzujian**
For #972 

Depend on PR #999

### CODE REVIEW

**Comment by zhangzujian** [comment]
Ready for review.

File: openstack/identity/v3/policies/requests.go [hunk 1]
```diff
@@ -99,3 +99,15 @@ func Create(client *gophercloud.ServiceClient, opts CreateOptsBuilder) (r Create
 	})
 	return
 }
+
+// Get retrieves details on a single policy, by ID.
+func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
```
Comment (jtopjian) [inline openstack/identity/v3/policies/requests.go]: Small nit: change `id` to `policyID` to match the other occurrences of `policyID`.

**Comment by zhangzujian** [comment]
@jtopjian ping for review

**Comment by zhangzujian** [comment]
recheck

**Comment by jtopjian** [comment]
LGTM - thank you :)

**Pull request merged**